SpectatorView/Compositor/UnityCompositorInterface/UnityCompositorInterface.vcxproj

```diff
@@ -91,7 +91,7 @@
   </PropertyGroup>
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|x64'">
     <LinkIncremental>false</LinkIncremental>
-    <IncludePath>$(KINECTSDK20_DIR)\inc;$(Canon_SDK)\EDSDK\Header;$(OpenCV_vc14)\..\..\include;$(IncludePath)</IncludePath>
+    <IncludePath>$(KINECTSDK20_DIR)\inc;$(Canon_SDK)\EDSDK\Header;$(OpenCV_vc14)\include;$(IncludePath)</IncludePath>

```

Comment:
This relative path is from the dependencies.props file.  Changing this will cause a breaking change for others using OpenCV